### PR TITLE
fix_#8589_increase_readability_of_scalar_newton_method

### PR DESCRIPTION
**Pull request by sudheerachary**
appropriate readability changes as mentioned in #8589   for the `newton method` in `scipy/scipy/optimize/zeros.py` are made.

### CODE REVIEW

File: scipy/optimize/zeros.py [hunk 1]
```diff
@@ -157,40 +157,37 @@ def newton(func, x0, fprime=None, args=(), tol=1.48e-8, maxiter=50,
         raise ValueError("tol too small (%g <= 0)" % tol)
     if maxiter < 1:
         raise ValueError("maxiter must be greater than 0")
+    p0 = 1.0 * x0
     if fprime is not None:
         # Newton-Rapheson method
         # Multiply by 1.0 to convert to floating point.  We don't use float(x0)
```
Comment (jaimefrio) [inline scipy/optimize/zeros.py]: This comment needs to move with `1.0 * p0`.

**Review by jaimefrio — COMMENTED** [review]
A small nit and this is ready to go.

**Review by jaimefrio — APPROVED** [review]

**Comment by sudheerachary** [comment]
the log for travis CI has ended up with `The job exceeded the maximum time limit for jobs, and has been terminated.` this message, what does this mean ? how to resolve ?

**Comment by person142** [comment]
> this message, what does this mean ? how to resolve ?

Travis limits the time for a job to 50 minutes, and there is some probability that we exceed it. I restarted the build.

**Comment by sudheerachary** [comment]
oh thanks.

**Comment by jaimefrio** [comment]
Thanks, @sudheerachary! In it goes.

**Pull request merged**